packages/react-native/src/theme/createReactNativeTheme.ts

```diff
@@ -0,0 +1,55 @@
+// Internal Style Dictionary methods
+import deepExtend from 'style-dictionary/lib/utils/deepExtend';
+import get from 'lodash/get';
+import mapValues from 'lodash/mapValues';
+
+import { defaultTheme } from './defaultTheme';
+import { ColorMode, Theme, ReactNativeTheme } from './types';
+import { Tokens } from './tokens';
+
+const mapValuesDeep = (v: object | string, callback: Function): object =>
+  typeof v == 'object'
+    ? mapValues(v, (v) => mapValuesDeep(v, callback))
+    : (callback(v) as object);
```

Comment:
Since we are not going to reuse the util outside this file, let's write it in a way where it handles typing the output for us so we don't need to cast on ln 45